src/client/firefox/create.js

```diff
@@ -54,12 +58,14 @@ export function createSource(
     isBlackBoxed: false,
     loadedState: "unloaded"
   };
+  clientCommands.registerSource(createdSource);
```

Comment:
fortunately i think we will simplify this pretty soon

https://github.com/devtools-html/debugger.html/pull/7365